policykit/policyengine/models.py

```diff
@@ -1026,6 +1027,143 @@ def __init__(self, *args, **kwargs):
         self.label_suffix = ''
 
 
+class CustomAction(models.Model):
```

Comment:
I wonder if a better name for this is CustomActionType. Since it's not really an action.